java-checks-testkit/src/test/java/org/sonar/java/checks/verifier/JavaCheckVerifierTest.java

```diff
@@ -93,7 +93,7 @@ public void verify_unexpected_issue() {
 
     try {
       JavaCheckVerifier.verify(FILENAME_ISSUES, visitor);
-      Fail.fail();
+      Fail.fail("");
```

Comment:
Can be replaced with http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion but maybe later